modules/rte_mis_school/rte_mis_school.links.task.yml

```diff
@@ -2,7 +2,7 @@
 rte_mis_school.school_registration.edit:
   route_name: rte_mis_school.school_registration.edit
   base_route: entity.user.canonical
-  title: Edit Profile
+  title: Register
```

Comment:
Can you confirm with Madhu, if school is registering for the first time it should be register or something more appropriate.
CC - @kunal-sin

Reply:
As per the feedback list, it is 'Register'. Again it's appropriate to confirm this from Madhu.